### DISCUSSION TITLE
Hard to finish a review

### DISCUSSION THREAD

**Discussion by TWiStErRob** [community/community#17885] [Mobile] [answered]
I left a comment on a specific line. Now I'm back at the PR screen and I see my comments are pending, and there's no way submit the review. I tried all the overflow menus and nothing.

**Comment by brianlovin** [comment, marked as answer]
@TWiStErRob at the top of the PR if you tap on the "Files changed" cell, then leave a review from that files changed view, it should publish all the pending review comments. Could you let me know if that works? I agree, this is not super clear or obvious today.

**Reply by TWiStErRob** [reply]
Ah, that's actually the same problem on Desktop web. Anyway, in the app I was expecting one of the top overflow menus (...) to contain a link to start submitting the review.

Additionally, probably a more intuitive one is to have the Comment button on PR screen be long-tappable where the Review button comes up as an option. Similarly on the file listing the Review button could be long-tappable to see or behave the Comment button.

**Reply by TWiStErRob** [reply]
Could there be a simple addition to make this simpler?

**Reply by TWiStErRob** [reply]
By the way, good to see the most relevant person (GithHub Mobile Designer) reply on the report! 👏

**Reply by brianlovin** [reply]
I've opened an issue on our end to make it more clear when you have pending review comments so that it's easier to navigate in and finish a review. Thanks!
